modules/apps/dispatch/dispatch-test/src/testIntegration/java/com/liferay/dispatch/service/test/DispatchTriggerLocalServiceTest.java

```diff
@@ -298,6 +298,43 @@ user, _getRandomDispatchExecutorType(),
 		}
 	}
 
+	@Test
+	public void testScheduleWithLaterCronExpressionSet() throws Exception {
+		DispatchTrigger dispatchTrigger = _addDispatchTrigger(
+			DispatchTriggerTestUtil.randomDispatchTrigger(
+				UserTestUtil.addUser(), _getRandomDispatchExecutorType(), 1));
+
+		DispatchTaskClusterMode dispatchTaskClusterMode =
+			DispatchTaskClusterMode.valueOf(
+				dispatchTrigger.getDispatchTaskClusterMode());
+
+		Calendar calendarStart = CalendarFactoryUtil.getCalendar(
```

Comment:
This does not follow the pattern. In Liferay, we usually call the variables ending with the type, for example `startCalendar`

![image](https://github.com/user-attachments/assets/d2cf9cab-b4da-43bd-bdd4-01cd3c0f8b58)